### ISSUE TITLE
Seems to have stopped working today

### ISSUE THREAD

**Issue by Shinmera** [SeinopSys/YTMySubs#7]
No idea what youtube changed this time around, but the extension no longer redirects to the subscriptions page automatically.

**Comment by SeinopSys** [comment]
Looks like another lovely A/B test that my account is not a part of, unfortunately I'm unlikely to be able to fix it until it started breaking for me as well. I'll keep this issue open until then, if you can share any additional details, please do.

**Comment by Shinmera** [comment]
Yeah, not sure what to even share about it. Fwiw it happens on all my machines, so not just a random setup thing. I'll keep you posted if it changes again, though.

**Comment by SeinopSys** [comment]
What usually helps narrow it down is the specific browser version, e.g. Mozilla Firefox 88.0.1 (64-bit), s well as the specific reproduction steps, such as "typing in youtube.com to the address bar and pressing enter loads the home page" or "clicking the youtube logo from a video page loads the home page". That's probably the most info you could help with without diving too deep into the developer tools.

**Comment by SeinopSys** [comment]
I have just installed Vivaldi version 3.8.2259.42 on my Windows 10 machine and the extension seems to work perfectly fine on there, I think this is unfortunately something on YouTube's side that they're experimenting with and I can't get it to happen for me, so waiting it is unfortunately.

**Comment by Krunklehorn** [comment]
I'm on Chrome Version 116.0.5845.141 (Official Build) (64-bit) and it just stopped working today.

Typing youtube.com into the address bar properly redirects to youtube.com/feed/subscriptions but the page is empty other than YouTube's search bar and the triple-bar on the left side.

Clicking on Home from the triple-bar menu has the same issue, but if click on Subscriptions it loads fine.

**Comment by SeinopSys** [comment]
I cannot reproduce the issue currently, if it also breaks for me I will look for a possible fix.

**Comment by Krunklehorn** [comment]
Turns out it was a temporary issue with uBlock origin. Apologies for the false alarm.

The plugin still works great! It's my most recommended. 👍
